monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -1017,6 +1167,46 @@ sealed abstract class Task[+A] extends Serializable {
   final def dematerialize[B](implicit ev: A <:< Try[B]): Task[B] =
     this.asInstanceOf[Task[Try[B]]].flatMap(fromTry)
 
+  /** Returns a new task that mirrors the source task for normal termination,
+    * but that triggers the given error on cancellation.
+    *
+    * Normally tasks that are cancelled become non-terminating.
+    * Here's an example of a cancelable task:
+    *
+    * {{{
+    *   val tenSecs = Task.sleep(10)
+    *   val task = tenSecs.fork.flatMap { fa =>
+    *     // Triggering pure cancellation, then trying to get its result
+    *     fa.cancel.flatMap(_ => fa)
+    *   }
+    *
+    *   task.timeout(10.seconds).runAsync
+    *   //=> throws TimeoutException
+    * }}}
+    *
+    * In general you can expect cancelable tasks to become non-terminating on
+    * cancellation.
+    *
+    * This `onCancelRaiseError` operator transforms a task that would yield
+    * [[Task.never]] on cancellation into one that yields [[Task.raiseError]].
+    *
+    * Example:
+    * {{{
+    *   import java.util.concurrent.CancellationException
+    *
+    *   val tenSecs = Task.sleep(10.seconds).onCancelRaiseError
```

Comment:
Forgotten parameter for `onCancelRaiseError`

Reply:
👍